addons/web/static/src/views/kanban/kanban_controller.js

```diff
@@ -191,6 +192,9 @@ export class KanbanController extends Component {
             }
         });
         this.searchBarToggler = useSearchBarToggler();
+        onMounted(() => {
+            this.firstLoad = false;
+        });
```

Comment:
```suggestion
        this.firstLoad = true;
        onMounted(() => {
            this.firstLoad = false;
        });
```
otherwise it only works because not setting it means it is undefined, and an undefined props is considered as a props that isn't given, which thus fallbacks on the default.